src/components/confirm/index.js

```diff
@@ -35,17 +39,17 @@ const connector = connect(
     }
 )
 
-const ConfirmWrapper = () => {
-    const { isVisible, ConfirmContentComponent, cancelHandler, confirmHandler, contentProps } = this.props;
+const ConfirmWrapper = (props) => {
+    const { isVisible, ConfirmContentComponent, handleCancel, handleConfirm, contentProps } = props;
```

Comment:
Previous naming was ok. The connector renames the props. see: https://github.com/KleeGroup/focus-components/blob/30cd3027e7124ce239eb866f7d1c3e399345e33a/src/components/confirm/index.js#L33